app/models/study.rb

```diff
@@ -912,24 +902,10 @@ def user_in_group_share?(user, *permissions)
     # check if api status is ok, otherwise exit without checking to prevent UI hanging on repeated calls
     if user.registered_for_firecloud && ApplicationController.firecloud_client.services_available?(FireCloudClient::SAM_SERVICE, FireCloudClient::RAWLS_SERVICE, FireCloudClient::THURLOE_SERVICE)
       group_shares = self.study_shares.keep_if {|share| share.is_group_share?}.select {|share| permissions.include?(share.permission)}.map(&:email)
-      # get user's FC groups
-      if user.access_token.present?
-        client = FireCloudClient.new(user:, project: FireCloudClient::PORTAL_NAMESPACE)
-      elsif user.api_access_token.present?
-        client = FireCloudClient.new
-        client.access_token[:access_token] = user.api_access_token
-      else
-        false
-      end
-      begin
-        user_groups = client.get_user_groups.map {|g| g['groupEmail']}
-        # use native array intersection to determine if any of the user's groups have been shared with this study at the correct permission
-        (user_groups & group_shares).any?
-      rescue => e
-        ErrorTracker.report_exception(e, user, { user_groups: user_groups, study: self.attributes.to_h})
-        Rails.logger.error "Unable to retrieve user groups for #{user.email}: #{e.class.name} -- #{e.message}"
-        false
```

Comment:
Good question - it isn't logged anymore.  I considered keeping it, but essentially all it will log are `401` for users not in compliance with the terms of service, or this very-corner-case `404`.  If you think either are worth keeping reporting for, I can put it back in.  Both felt non-actionable to me, hence why I skipped reporting.

Reply:
Makes sense -- I appreciate the explanation.